src/vehicle.cpp

```diff
@@ -4591,9 +4591,14 @@ cata::optional<vehicle_stack::iterator> vehicle::add_item( int part, const item
     }
     // const int max_weight = ?! // TODO: weight limit, calculation per vpart & vehicle stats, not a hard user limit.
     // add creaking sounds and damage to overloaded vpart, outright break it past a certain point, or when hitting bumps etc
+    vehicle_part &p = parts[ part ];
+    if( p.is_broken() ) {
+        //debugmsg( "Tried to add item (%s) to a broken part (%s)", itm.tname(), p.name() );
```

Comment:
Please don't commit commented out code. Either it's useful code and should be run, so it should not be a comment at all. Or it is useless code that should not be there in the first place.

Reply:
Fixed. 
Thanks!